### PR TITLE
Fix singlestat panel to work with Grafana 9

### LINKED ISSUE #2: Fix grafana-singlestat-panel plugin and update description in catalog
Originated from: https://github.com/grafana/singlestat-panel/issues/3

**Why?**
https://grafana.com/grafana/plugins/singlestat/ was removed in Grafana 8 as the [stat panel](https://grafana.com/docs/grafana/latest/visualizations/stat-panel/) was introduced including automatic migration towards using it.

For those users that still want keep the old panel we have created an alternative - https://github.com/grafana/singlestat-panel plugin. This alternative has not been maintained for a while and is not compatible anymore with Grafana 9.

**Goal**
- Get it working with Grafana 9 and publish a new version. Most likely it should be enough to switc from using `getColorForTheme` to `getColorByName`
- Do not migrate it from Angular to React

**Additionally**
- Update https://grafana.com/grafana/plugins/singlestat/ documentation or redirect it to https://grafana.com/grafana/plugins/grafana-singlestat-panel/
- Update [README.md](https://github.com/grafana/singlestat-panel/blob/main/README.md) to point towards more up-to-date documentation

### PR DESCRIPTION
**Pull request by tolzhabayev**
Fixes: #2 #3 

```
- Bump dependencies
- Migrate to using `getColorByName` instead of `getColorForTheme` to support Grafan 9
- Migrate to using `GrafanaThema2` instead of `GrafanaTheme`
- Migrate to using `convertOldAngularValueMappings` from `@grafana/data` package instead of `convertOldAngularValueMapping` from `@grafana/ui` package
- Bump minimum supported Grafana version to `8.+`
- Remove dependency for `@grafana/ui` as not needed any more
```

When this change is published this PR might be obsolete: https://github.com/grafana/grafana/pull/51149 

This PR has to be merged and published first: https://github.com/grafana/singlestat-panel/pull/5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "grafana-singlestat-panel",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (leventebalogh) [inline package.json]: **Question:** as supported Grafana version has changed and is not supporting `7.x` anymore, should we do a major version bump for the plugin as well (to `2.0.0`), due to this being a "breaking change"?

**Review by leventebalogh — APPROVED** [review]
LGTM, nice! 👏

File: package.json (hunk 1 above)
Comment (tolzhabayev) [reply package.json]: Good point @leventebalogh, switched to v2.0.0

**Review by briangann — APPROVED** [review]
lgtm

**Pull request merged**